### DISCUSSION TITLE
how to train a model for multiple faces?

### DISCUSSION THREAD

**Discussion by Art4DD** [d8ahazard/sd_dreambooth_extension#296] [Q&A]
how to train one model for several persons at once?

I want to give each face a unique instance promt to control the face in the picture. I need some kind of tutorial or at least a description of how to do this.
I assume that me need to use the description of the images [filewords], but I need some documentation to understand how to solve my problem.

**Comment by d8ahazard** [comment]
Take a look at the "concepts.json" file in the extension's /dreambooth folder for an example of how to train multiple concepts at once.

Or, wait a few days, and I'll have multiple concept support added natively to the UI, no need for mucking around with JSON if you don't want to.

**Reply by Art4DD** [reply]
Did I get the idea right?
There are two different folders for two different persons.

> [
>     {
>         "instance_prompt":      "aiface1 face",
>         "class_prompt":         "face",
>         "sample_prompt":        "aiface1 face",
>         "negative_prompt":      "",
>         "instance_data_dir":    "../../../data/aiface1",
>         "class_data_dir":       "../../../data/class_img"
>     },
>     {
>         "instance_prompt":      "aiface2 face",
>         "class_prompt":         "face",
>         "sample_prompt":        "aiface2 face",
>         "negative_prompt":      "",
>         "instance_data_dir":    "../../../data/aiface2",
>         "class_data_dir":       "../../../data/class_img"
>     }
> ]

But what if I want their style to be common?
Let's say I have 100 images in the same style. Of these, 10 images of one person and 10 images of another person.
How do I write down the concept list correctly?
This?

> [
>     {
>         "instance_prompt":      "aistyle123 style",
>         "class_prompt":         "style",
>         "sample_prompt":        "aistyle123 style",
>         "negative_prompt":      "",
>         "instance_data_dir":    "../../../data/aistyle123",
>         "class_data_dir":       "../../../data/class_img"
>     },
>     {
>         "instance_prompt":      "aiface1 face",
>         "class_prompt":         "face",
>         "sample_prompt":        "aiface1 face",
>         "negative_prompt":      "",
>         "instance_data_dir":    "../../../data/aiface1",
>         "class_data_dir":       "../../../data/class_img"
>     },
>     {
>         "instance_prompt":      "aiface2 face",
>         "class_prompt":         "face",
>         "sample_prompt":        "aiface2 face",
>         "negative_prompt":      "",
>         "instance_data_dir":    "../../../data/aiface2",
>         "class_data_dir":       "../../../data/class_img"
>     }
> ]

**Comment by MV799** [comment]
Pardon me, but how does class trigger and instance trigger fit into this example?
